doc/ChangeLog

```diff
@@ -1,3 +1,43 @@
+===============================================================
+
+Tag name:
+Originator(s): jimmielin
+Date: August 22, 2024
+One-line Summary: tropopause_find CCPP-ization
+Github PR URL: https://github.com/ESCOMP/atmospheric_physics/pull/112
+
+This PR fixes the following NCAR/atmospheric_physics Github issues:
+    - Creates the CCPP interface for the tropopause_find routine
+
+    Several (extensive) changes were made to tropopause_find.F90 for CCPP-compatibility:
+    - tropopause_read_file, which provides tropp_p_loc and tropp_days climatological data used as the default fall-back method, has been moved to a utility module in CAM-SIMA to read, regrid, and provide this climatology to the CCPP-ized physics; it also removes lchnk indexing since they're no longer used in CAM-SIMA.
+    - tropopause_find used to accept (optional) arguments for returning tropT, tropZ, tropP (tropopause temperature, height, and pressure) which are no longer optional;
+    - tropopause_find accepted "primary" and "backup" methods for finding the tropopause. All the methods known to be used in CAM are now calculated at the same time in the tropopause_find_run main driver routine; it calls the underlying logic as appropriate with and populates the appropriate standard name physics fields. Because the same physical quantities for tropopause level, height, pressure, temperature, etc. using different methods are used throughout the CAM physics and simultaneously output in the diagnostics, the same standard names with a suffix are used to differentiate between these quantities computed by different methods.
+    - Standard CCPP-ization procedures, metadata, explicit use of fields instead of physics state, only passing active :ncol fields, ...
+
+    A "shim" that will completely replicate existing behavior, bit-for-bit in current CAM, will be included in current CAM as tropopause.F90.
```

Comment:
We typically use "interface code" in CAM to signify what I believe you are conveying with the word "shim".  If that is equivalent, then there was at least one other place where I saw the word "shim"

Reply:
Thank you, updated with "interface code". I believe the other place was in the ESCOMP/CAM pull request, which I've now updated as well.